CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Added
 
 - Synchronize scroll with active transcript sentence
+- Expose origin playlist when resource come from a portable one
+
+### Fixed
+
+- Fix how permissions are used to display dashboard
```

Comment:
Can you be more specific?

Reply:
The explanation is in the commit message. It is not enough ?

Reply:
Maybe:
```
- Fix dashboard read versus update permissions in situations of playlist portability
```